src/app/services/internationalPayment/international-payment.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from "@angular/core";
+import { InternationalPayment } from "src/app/models";
+import { Observable } from "rxjs";
+import { HttpService } from "../http.service";
+import { HttpMethods } from 'src/app/config';
```

Comment:
move all constants to the `environments` file

Reply:
@malep2007 done